src/HUDL/HUDL.cpp

```diff
@@ -12,14 +12,16 @@
 #include <HUDL/HUDL.hpp>
 #include <cstdio>
 #include <cstring>
-#include "EVT/dev/BitmapFonts.hpp"
+
 // clang-format on
 
 namespace IO = EVT::core::IO;
 namespace DEV = EVT::core::DEV;
 namespace log = EVT::core::log;
 
 namespace HUDL {
+typedef DEV::LCD::FontSize FontSize;
```

Comment:
This is a nice solution, but we need to be careful only to do this in `.cpp` files. If we do this in header files, the typedef could get applied places we don't want it to be